packages/aws-lambda-auto-wrap/test/cjs_test.js

```diff
@@ -0,0 +1,192 @@
+/*
+ * (c) Copyright IBM Corp. 2022
+ */
+
+'use strict';
+
+const sinon = require('sinon');
+const expect = require('chai').expect;
+const instana = require('@instana/aws-lambda');
+
+describe('cjs wrapper', function () {
```

Comment:
_praise:_ Kudos for adding tests for the original commonjs wrapper! 🙇

Reply:
I also ran the test against the original index.js handler from main branch. Looks good 👍